.github/workflows/base.yml

```
@@ -60,8 +60,11 @@ jobs:
         timeout_minutes: 120
         command: docker buildx build -f Dockerfile.base.ubuntu-bionic -t ${ORG}/github-runner-base:ubuntu-bionic --output "type=image,push=true" --platform linux/amd64,linux/arm/v7,linux/arm64 .
 
-  debian_base_buster_deploy:
+  debian_base_deploy:
     runs-on: ubuntu-latest
+    strategy:
```

```suggestion
    strategy:
    fail-fast: false
```
If you put multiple runs in a file, it makes sense for it to continue with the second one if the first one fails.

This especially makes sense if you are running Debian unstable as it will randomly break.

---

Good point, fixed in f0f4f7d8bce1b4fb30cbc8816f5133dc45824496